statick

```diff
@@ -24,6 +24,9 @@ def main():
 
     path = parsed_args.path
     issues, success = statick.run(path, parsed_args)
+    for tool in issues:
```

Comment:
This is the bit that causes Statick to return an exit code if any of the tool plugins found issues. Prior to this we would not get any notification that problems existed unless something bigger happened, like a tool not being found. The result was that issues could exist but Travis CI would show successful builds.